Parsing_onliner_project/httpRequests.py

```diff
@@ -0,0 +1,23 @@
+"""Storage module for class HTTPClient"""
```

Comment:
Folders and files could not contain with Capital letters.
It should be named as "clients" and file itself should be "http_client.py"